qbraid/transpiler/conversions/pennylane/pennylane_to_qasm2.py

```diff
@@ -35,4 +35,4 @@ def pennylane_to_qasm2(tape: QuantumTape) -> Qasm2StringType:
         str: OpenQASM 2.0 representation of the tape
 
     """
-    return tape.to_openqasm()
+    return tape.to_openqasm(measure_all=False)
```

Comment:
Will this implicitly remove measurement gates from a quantum tape if they already exist? If so, may want to account for that so we don't remove measurements from a circuit without the user knowing

Reply:
So what [`measure_all`](https://docs.pennylane.ai/en/stable/code/api/pennylane.tape.QuantumTape.html?highlight=to_openqasm#pennylane.tape.QuantumTape.to_openqasm) does is that it adds measurements to the qubits which do not have an output and does not remove the user specified measurements. 

Since this was by default set to `True`, set it as `False` here so that circuit is not modified